packages/iris-grid/src/sidebar/TableCsvExporter.jsx

```diff
@@ -226,15 +226,21 @@ class TableCsvExporter extends Component {
             value={downloadOption}
             disabled={isDownloading}
           >
-            <RadioItem value={TableCsvExporter.DOWNLOAD_OPTIONS.ALL_ROWS}>
+            <RadioItem
+              value={TableCsvExporter.DOWNLOAD_OPTIONS.ALL_ROWS}
+              dataTestid="csv-exporter-download-all-radio"
```

Comment:
As mentioned above prefer the input type to be first in the naming convention, similar to how bootstrap suggests: https://codeguide.co/#css-classes